### DISCUSSION TITLE
Are there any plans to make this package work with Laravel Herd?

### DISCUSSION THREAD

**Discussion by danielh-official** [spatie/laravel-pdf#186] [Q&A]
If I start a server the traditional way, then this package works fine.

If I do it via Laravel Herd, however, all exported PDFs end up being blank.

Is there an existing fix for this or will that be addressed in a later release?

**Comment by ArielMejiaDev** [comment]
I am using Herd and it works without any issue

**Reply by danielh-official** [reply]
> I am using Herd and it works without any issue

Do you have any custom configuration on Herd. I have the default setup with PHP 8.3. Also, I am using Node v18.20.3. Other than that, I use the PDF export in the standard way. I have the code in a `DownloadController` that's tied to a GET route in the web routes.
